docs/02-app/01-building-your-application/03-rendering/03-edge-and-nodejs-runtimes.mdx

```diff
@@ -27,6 +27,7 @@ There are many considerations to make when choosing a runtime. This table shows
 | Security                                                                                                                                            | Normal | High       | High             |
 | Latency                                                                                                                                             | Normal | Low        | Lowest           |
 | npm Packages                                                                                                                                        | All    | All        | A smaller subset |
+| Revalidation                                                                                                                                        | Yes    | Yes        | No               |
```

Comment:
```suggestion
| Prerendering                                                                                                                                        | Yes    | Yes        | No               |
```

Should we say prerendering here specifically as fetch cache revalidate does work